services/src/test/scala/cromwell/pipeline/service/ProjectConfigurationServiceTest.scala

```diff
@@ -102,19 +124,31 @@ class ProjectConfigurationServiceTest extends AsyncWordSpec with Matchers with M
       val updateResult: Unit = ()
 
       "return complete status for deactivating configuration" in {
-        when(configurationRepository.getById(projectId)).thenReturn(Future.successful(Some(activeConfiguration)))
+        when(projectService.getUserProjectById(projectId, userId)).thenReturn(Future(dummyProject))
+        when(configurationRepository.getConfigurationById(projectId))
+          .thenReturn(Future.successful(Some(activeConfiguration)))
         when(configurationRepository.updateConfiguration(inactiveConfiguration))
           .thenReturn(Future.successful(updateResult))
-        configurationService.deactivateConfiguration(projectId).map(_ shouldBe updateResult)
+        configurationService.deactivateConfiguration(projectId, userId).map(_ shouldBe updateResult)
       }
 
       "return exception if no configuration was matched" in {
-        when(configurationRepository.getById(projectId)).thenReturn(Future.successful(None))
+        when(projectService.getUserProjectById(projectId, userId)).thenReturn(Future(dummyProject))
+        when(configurationRepository.getConfigurationById(projectId)).thenReturn(Future.successful(None))
         configurationService
-          .deactivateConfiguration(projectId)
+          .deactivateConfiguration(projectId, userId)
           .failed
           .map(_ should have.message("There is no project to deactivate"))
       }
+
+      "return exception if user is not owner for project" in {
```

Comment:
Maybe change wording a little to something like "return exception if user is not the project owner"

Reply:
Ok. I will change it.

Reply:
Changed.